packages/nocodb/src/models/User.ts

```
@@ -213,12 +226,18 @@ export default class User implements UserType {
+    const user = await ncMeta.metaGet2(
       RootScopes.ROOT,
       RootScopes.ROOT,
       MetaTable.USERS,
       userRefreshToken.fk_user_id,
     );
+
+    if (user?.is_deleted) {
+      return null;
+    }
+
+    return this.castType(user);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)